dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -149,13 +149,8 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
         }
         catch (...)
         {
-            HRESULT hrError = wil::ResultFromCaughtException();
-            PushNotificationTelemetry::ChannelRequestedByApi(
-                hrError,
-                AppModel::Identity::IsPackagedProcess(),
-                remoteId);
-
-            THROW_HR(hrError);
+            PushNotificationTelemetry::ChannelRequestedByApi(wil::ResultFromCaughtException(), AppModel::Identity::IsPackagedProcess(), remoteId);
+            throw;
```

Comment:
Why throw a new exception. Can we rethrow instead?

Reply:
This was from @jonwis guidance here:

https://github.com/microsoft/WindowsAppSDK/pull/1415#discussion_r719935295

Reply:
Ah ok. Gotcha